### ISSUE TITLE
Recommended chat duration

### ISSUE THREAD

**Issue by cyrusstoller** [cyrusstoller/EchoRemix#9]
Per @sashenka87's question in #3, what is a good recommendation for the desired chat duration. Maybe this is something to add to the FAQs? It's easy to add a notification that you've talked for say 5 minutes and that you can consider moving onto the next conversation. My concern is that this may inadvertently cut some good conversations short.

I'm open to suggestions.

**Comment by ezrastoller** [comment]
I think giving an idea of how long a typical conversation lasts would be helpful! It could be a helpful way to encourage people to give conversations the time to develop.

**Comment by cyrusstoller** [comment]
Almost all of my conversations have lasted >10 minutes (assuming there was at least one back and forth). Is that consistent with your experience?

**Comment by megfitz** [comment]
Yes but I find they rarely last that long on the topic set and once you move off the topic it's easier to get into less productive chat territory.

------------
Meaghan Fitzgerald
650-799-8995

Sent on the go, apologies for any typos.

**Comment by cyrusstoller** [comment]
Maybe there should be a system alert to consider changing topics after a conversation has gone 10 minutes?
